src/components/SearchBar.js

```diff
@@ -0,0 +1,60 @@
+import { Box, InputAdornment, TextField } from '@mui/material';
+import Image from 'next/image';
+import { useAuthContext } from '~/contexts/hooks';
+import theme from '~/theme';
+
+export default function SearchBar() {
+  const { session } = useAuthContext();
+  return (
+    <Box sx={{ display: 'flex', alignItems: 'center' }}>
+      <TextField
+        variant="outlined"
+        placeholder={`Xin chào ${session?.firstName}, bạn cần gì?`}
+        InputProps={{
+          startAdornment: (
+            <InputAdornment position="start">
+              <Image
+                src="/images/search-icon.svg"
+                width={30}
+                height={30}
+                alt="search-icon"
+                style={{
+                  cursor: 'pointer',
+                  margin: '0 0 0 10px',
+                  color: theme.palette.aiTextColor,
```

Comment:
use theme hook in this case `style` is correct, but try to use `sx` for MUI component to access the theme directly

Reply:
nhưng cái thằng Image đến từ next mà anh

Reply:
ok, thế đổi lại vụ import là đc